docs/products/kafka/howto/get-topic-partition-details.rst

```diff
@@ -0,0 +1,46 @@
+Get partition details of a Kafka topic
+==============================================
+
+Use one of three available approaches to retrieve partition details of a Kafka topic.
+
+With the help of Aiven Console
+-------------------------------
+
+In the Aiven Console follow these steps:
+
+1. Log in to the Aiven web console and select your Apache Kafka service.
+2. On the **Topics** tab click the three dots next to the topic you wish to inspect.
+3. In the appeared menu click on the **Info** entry, it will open a modal window with topic details.
+4. Navigate to the **Partitions** tab.
```

Comment:
We can also state that if they want to check the data, they need to enable the REST APIs from the service overview

Reply:
I wonder, if this fact is not relevant to the topic of this article? Or is it?

Reply:
Let's remove it, yeah I agree, no need to add it here